src/BenchmarkDotNet/Toolchains/MonoWasm/WasmToolchain.cs

```diff
@@ -1,40 +1,46 @@
-﻿using BenchmarkDotNet.Toolchains.DotNetCli;
-using JetBrains.Annotations;
+﻿using System.Collections.Generic;
 using BenchmarkDotNet.Characteristics;
-using BenchmarkDotNet.Running;
-using BenchmarkDotNet.Loggers;
 using BenchmarkDotNet.Portability;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.DotNetCli;
+using BenchmarkDotNet.Validators;
+using JetBrains.Annotations;
 
 namespace BenchmarkDotNet.Toolchains.MonoWasm
 {
     [PublicAPI]
-    public class WasmToolChain : Toolchain
+    public class WasmToolchain : Toolchain
```

Comment:
Renaming public types is considered to be a breaking change and we avoid doing that if we can. However, this particular type is most likely not used directly by anyone, so it's OK-ish ;)